src/test/java/io/snyk/jenkins/jcasc/ConfigurationAsCodeTest.java

```diff
@@ -23,16 +23,16 @@ public class ConfigurationAsCodeTest {
 
   @Test
   @ConfiguredWithCode("configuration-as-code.yml")
-  public void should_support_configuration_as_code() throws Exception {
+  public void should_support_configuration_as_code() {
     SnykInstallation.SnykInstallationDescriptor descriptor = ExtensionList.lookupSingleton(SnykInstallation.SnykInstallationDescriptor.class);
-    assertEquals(descriptor.getInstallations().length, 1);
+    assertEquals(1, descriptor.getInstallations().length);
 
     SnykInstallation installation = descriptor.getInstallations()[0];
     assertEquals("snyk", installation.getName());
     DescribableList<ToolProperty<?>, ToolPropertyDescriptor> properties = installation.getProperties();
     DescribableList<ToolInstaller, Descriptor<ToolInstaller>> installers = ((InstallSourceProperty) properties.get(0)).installers;
     assertEquals(PlatformItem.LINUX, ((SnykInstaller) installers.get(0)).getPlatform());
-    assertEquals(Long.valueOf(36l), ((SnykInstaller) installers.get(0)).getUpdatePolicyIntervalHours());
+    assertEquals(Long.valueOf(36L), ((SnykInstaller) installers.get(0)).getUpdatePolicyIntervalHours());
```

Comment:
Code cleanup, see https://docs.oracle.com/javase/specs/jls/se8/html/jls-3.html#jls-3.10.1

```The suffix L is preferred, because the letter l (ell) is often hard to distinguish from the digit 1 (one). ```